RELEASE/scripts/postsool.ash

```diff
@@ -748,6 +760,32 @@ void handlePostAdventure()
 				buffMaintain($effect[Ceaseless Snarling], 0, 1, 10);
 			}
 		}
```

Comment:
Surely this stuff should go in presool?

Reply:
Since I didn't initially put it there, I really don't want to move it lest it break something. If someone else who knows what it impacts wants to do it then they can.

Reply:
Sure, don't worry about it in this PR and soolar or I can go through and fix it later.